api-gateway-routes/api_gateway_proxy_function.rb

```diff
@@ -36,7 +33,7 @@ def on_connect(event, context)
     :queueUrl => sqs_queue.queue_url,
     :apiEndpoint => api_endpoint,
     :connectionId => request_context["connectionId"],
-    :projectUrl => project_url,
+    :projectUrl => request_context["authorizer"]["project_url"],
```

Comment:
currently the token does not contain the project_url--do we need to add it? It does contain the channel_id

Reply:
For now, at least, I'd like to pass in the project_url here. I've got a follow-up PR on dashboard to do that. There's pretty much one place where we build the URLs for accessing the S3 API, so if we ever update it, I'd like to get that automatically in javabuilder.